hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -625,7 +625,12 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
     Collection<String> omAdminGroups =
         OzoneConfigUtil.getOzoneAdminsGroupsFromConfig(configuration);
     LOG.info("OM start with adminUsers: {}", omAdminUsernames);
-    omAdmins = new OzoneAdmins(omAdminUsernames, omAdminGroups);
+    // Get super read list
+    Collection<String> omSuperReadUsernames =
+            OzoneConfigUtil.getOzoneSuperReadUserNamesFromConfig(configuration);
+    Collection<String> omSuperReadGroups =
+            OzoneConfigUtil.getOzoneSuperReadGroupsFromConfig(configuration);
+    omAdmins = new OzoneAdmins(omAdminUsernames, omAdminGroups, omSuperReadUsernames, omSuperReadGroups);
```

Comment:
Instead of duplicating logic in `OzoneAdmins` to handle 2 pairs of lists, it would be better to create two instances of `OzoneAdmins`:

```
omAdmins = new OzoneAdmins(omAdminUsernames, omAdminGroups);
readOnlyAdmins = new OzoneAdmins(omSuperReadUsernames, omSuperReadGroups);
```

and use `readOnlyAdmins.isAdmin()` for the `isSuperRead` check.

Reply:
okey, i will do this, thanks